.github/actions/setup_dbt/action.yaml

```diff
@@ -14,7 +14,7 @@ runs:
       uses: ./.github/actions/load_environment_variables
 
     - name: Configure AWS credentials
-      uses: aws-actions/configure-aws-credentials@v2
+      uses: aws-actions/configure-aws-credentials@v4
```

Comment:
I figured we might as well bump this action version since we started getting deprecation warning for v2 in the log output. Also v4 came out [just a few days ago](https://github.com/aws-actions/configure-aws-credentials/releases/tag/v4.0.0) 🤷